Common/IRI.hs

```diff
@@ -921,3 +922,9 @@ mergeCurie c i =
 
 deleteQuery :: IRI -> IRI
 deleteQuery i = i { abbrevQuery = "" }
+
+addSuffixToIRI :: String -> IRI -> IRI
+addSuffixToIRI s i = if not $ null $ abbrevQuery i 
+                   then i{abbrevQuery = abbrevQuery i ++ s}
+                  else  
+                        i{abbrevPath  = abbrevPath i ++ s}
```

Comment:
please have "then" and "else" equally indented, and please delete the line break after "else"

Reply:
Done